src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -16,9 +16,11 @@
 #include <string>
 #include "Header Files/CalcEngine.h"
 #include "Header Files/CalcUtils.h"
+#include "NumberFormattingUtils.h"
 
 using namespace std;
 using namespace CalcEngine;
+using namespace CalcManager::NumberFormattingUtils;
```

Comment:
Generally, we should maintain the CalcManager as a layer above the CalcEngine, and keep the engine from taking dependencies on the manager.